### DISCUSSION TITLE
请添加一个中国胃但是身体在美国的做饭指南

### DISCUSSION THREAD

**Discussion by diaohaiyao** [Anduin2017/HowToCook#1318] [Ideas]
如题：人在美丽国小村，物价不低。
楼下有超市，但是是欧美超市，多是老白男和墨西哥人使用的食材。
那么如何用这些东西，在足够性价比下（刀和时间）满足一个中国胃呢。我和我老婆很急~~~

**Comment by blackpatton17** [comment]
import std::root_veggies;

**Comment by dingding71** [comment]
需要什么食材，自己new就完事了

**Comment by altpend** [comment]
这也太简单了, 中餐不就是各种调味料, 想当年我天天买意面做葱花面, 肉这些不都是通用的,有什么做什么喽, 蔬菜区该有的基本也都有, 会做饭随便整

**Comment by Xxxxxxyz** [comment]
中超

**Comment by Xyncgas** [comment]
有些还是能够买得到吧比如沃尔玛有肉蛋奶，买番茄做炒蛋，买面粉做饼干，牛奶直接喝，不想做饭那就方便面然后一根烟

**Comment by Nacccy** [comment]
主要是香料调味料这些吧，可以考虑网购在家里囤一点，其他蔬菜肉蛋奶沃尔玛都有
